code/modules/awaymissions/mission_code/ruins/telecomns.dm

```diff
@@ -391,6 +438,18 @@ GLOBAL_LIST_EMPTY(telecomms_trap_tank)
 		/datum/nothing = 4,
 		/obj/item/melee/baton/cattleprod/teleprod = 1)
 
+/obj/effect/spawner/random_spawners/telecomms_weldertank_maybe
+	name = "weldertank maybe"
+	result = list(
+		/datum/nothing = 3,
+		/obj/structure/reagent_dispensers/fueltank = 1)
+
+/obj/effect/spawner/random_spawners/telecomms_doomba_one_in_twenty
+	name = "weldertank maybe"
+	result = list(
+		/datum/nothing = 19,
+		/mob/living/simple_animal/bot/secbot/buzzsky/telecomms/doomba = 1)
+
```

Comment:
![image](https://github.com/user-attachments/assets/c96d1e49-d831-4357-84af-95e0dea08088)

Reply:
hehehehe, no-
Oh wait I see what he wants. I read this as spawn 25 welder tanks when I first read it lmao

Reply:
Lmfao, happens